translator/conventions/vendorconventions/aws.go

```diff
@@ -0,0 +1,36 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vendorconventions
```

Comment:
`awsconventions`? What do you think?

Reply:
Yeah I considered that, but I don't think we'll need more than one file for defining AWS conventions. If it's ok to just have a package with a single file I'll rename. Alternatively I can just move `aws.go` to `conventions` and do away with the subpackage altogether.

Reply:
I would expect to have only aws/azure/gcp, not too many...

Reply:
Ok, so I'll get rid of the subpackage and just move `aws.go` to conventions package, since even with the other cloud providers it won't get too crowded.